lib/Monads/nondet/Nondet_While_Loop_Rules.thy

```diff
@@ -618,7 +618,7 @@ lemma whileLoopE_liftE:
    apply (rule set_eqI, rule iffI)
     apply clarsimp
     apply (clarsimp simp: in_liftE whileLoop_def)
-    \<comment> \<open>The schematic existential is instantiated by 'subst isr_Inr_proj' ... 'rule refl' in two lines\<close>
+    \<comment> \<open>The schematic existential is instantiated by @{text "subst isr_Inr_projr' ... rule refl"} in two lines\<close>
```

Comment:
The `'` can be removed as well, it's left over from how the comment used to be formatted.
```suggestion
    \<comment> \<open>The schematic existential is instantiated by @{text "subst isr_Inr_projr ... rule refl"} in two lines\<close>
```

Reply:
ah, yes